packages/language-server/test/snapshots/xml-view-diagnostics/snapshots-utils.ts

```diff
@@ -47,7 +53,7 @@ export function getExpectedDiagnosticsLSPResponse(
 const ui5ModelPromise = generateModel({ version: "1.71.14" });
 let ui5Model: UI5SemanticModel | undefined = undefined;
 
-export async function getActualDiagnosticLSPResponse(
+export async function computeNewDiagnosticLSPResponse(
```

Comment:
I preferred the previous name (with `actual`) since you actually call the API here

Reply:
I too normally prefer the names with `actual|expected` prefixes in test cases.

However I felt there was a bit of confusion here, (perhaps due to the input **also** being used as a snapshot?). So I selected terms closer to the snapshots testing methodology concepts (e.g: `newlyComputed` vs `snapshot`) and tried using those terms **consistently** everywhere in this file.

Reply:
Ok